src/components/menu/MenuOpener/MenuOpenedByHover.tsx

```diff
@@ -0,0 +1,54 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import { IMenuOpenedBySharedProps } from './MenuOpenedBySharedProps';
+import MenuPosition from '../Position/MenuPosition';
+
+export default class MenuOpenedByHover extends React.Component<IMenuOpenedBySharedProps, {}> {
+    private static openCloseDelayMs = 200;
+
+    private timer: number = null;
```

Comment:
Please name the time so it's clear what's the purpose of it.

Reply:
Renamed to `timerCloseDelay`